/**:
  ros__parameters:

    # adap2e charateristics
    base:
      wheelbase: 1.38
      inertia:
        mass: 340.
        center: [0.6, 0.0, 0.7]
        z_moment: 259
      command_limits:  
        minimal_longitudinal_speed: -2.0
        maximal_longitudinal_speed: 2.0
        maximal_front_steering_angle: 0.7
        maximal_rear_steering_angle: 0.7

    # configuration of longitudinal control
    longitudinal_control:
      minimal_linear_speed: 0.3

    # configuration for classic and predictive lateral control laws
    lateral_controls:
      selected: classic 
      classic:
        gains:
          front_kp: 0.4
          rear_kp: 0.5
      predictive:
        gains:
          front_kp: 0.4
          rear_kp: 0.7
        prediction:
          horizon: 10.
          a0: 0.1642
          a1: 0.1072
          b1: 1.0086
          b2: -0.2801

    # control pointset
    pointset:
      desired_linear_speed: 1.0
      desired_lateral_deviation: 0.0

    #configuration for cmd_mux auto connection
    cmd_output:
      message_type: romea_mobile_base_msgs/TwoAxleSteeringCommand
      priority: 10
      rate: 10.


